src/DonationForms/resources/registrars/templates/fields/File.tsx

```diff
@@ -0,0 +1,25 @@
+import type {FieldProps} from '@givewp/forms/propTypes';
+
+export default function File({Label, ErrorMessage, fieldError, placeholder, inputProps}: FieldProps) {
+    const {setValue} = window.givewp.form.hooks.useFormContext();
+    const {name} = inputProps;
+
+    return (
+        <label>
+            <Label />
+
+            <input
+                type="file"
+                aria-invalid={fieldError ? 'true' : 'false'}
+                placeholder={placeholder}
+                onChange={(e) => {
+                    setValue(name, e.target.files[0]);
+                }}
+            />
```

Comment:
I wouldn't be opposed to using something like [React Dropzone](https://react-dropzone.js.org/) as a nice, new UI.

Reply:
Yeah we've used that in the past and it sure was nice.  I'd be cool with circling back on the actual file field UI if people would prefer that - or maybe it's actually a form field manager setting